.changeset/twelve-tigers-stare.md

```diff
@@ -0,0 +1,274 @@
+---
+"generated-test-app": patch
+"rainbowkit-next-app": patch
+"with-next-siwe-iron-session": patch
+"with-next-rainbow-button": patch
+"with-next-siwe-next-auth": patch
+"with-next-custom-button": patch
+"with-next-wallet-button": patch
+"with-create-react-app": patch
+"with-next-app-i18n": patch
+"with-next-mint-nft": patch
+"@rainbow-me/create-rainbowkit": patch
+"@rainbow-me/rainbow-button": patch
+"with-next-app": patch
+"with-remix": patch
+"@rainbow-me/rainbowkit": patch
+"with-next": patch
+"with-vite": patch
+"example": patch
+"site": patch
+---
+
+**Wagmi V2 Integration and EIP6963 Support**
+
+Upgraded to wagmi v2 and introduced support for EIP6963.
+
+EIP6963 wallets are now automatically placed in the `Installed` section of the RainbowKit connect modal. All projects within the RainbowKit repository examples folder have been updated to work with wagmi v2.
+
+**Before:**
+
+```tsx
+import "@rainbow-me/rainbowkit/styles.css";
+import { getDefaultWallets, RainbowKitProvider } from "@rainbow-me/rainbowkit";
+import { configureChains, createConfig, WagmiConfig } from "wagmi";
+import { mainnet } from "wagmi/chains";
+import { publicProvider } from "wagmi/providers/public";
+
+const { chains, publicClient, webSocketPublicClient } = configureChains(
+  [mainnet],
+  [publicProvider()]
+);
+
+const { connectors } = getDefaultWallets({
+  appName: "RainbowKit demo",
+  projectId: "YOUR_PROJECT_ID",
+  chains,
+});
+
+const wagmiConfig = createConfig({
+  autoConnect: true,
+  connectors,
+  publicClient,
+  webSocketPublicClient,
+});
+
+const App = () => {
+  return (
+    <WagmiConfig config={wagmiConfig}>
+      <RainbowKitProvider chains={chains}>{/* Your App */}</RainbowKitProvider>
+    </WagmiConfig>
+  );
+};
+```
+
+**After:**
```

Comment:
Checkout this markdown formatting for diffs: https://github.com/wevm/wagmi/releases/tag/wagmi%400.12.0